index.html

```diff
@@ -175,33 +198,29 @@ <h1>Multi-Post Stories</h1>
                 <li><h5>JavaScript</h5></li>
             </ul>
             <div class="button item"><h3><a>See Project</a></h3></div>
+        </div>
         </li>
     </ul>
     </section>
     <section class="work_grid">
     <ul>
-        <li class="box">
+        <li class="box_about">
+            <div class="box_about_bg"></div>
+            <div class="sub_box">
             <div class="title item">
                 <h1>About Myself</h1>
             </div>
-            <ul class="features item">
-                <li><h4 class="feature_1">CANOPY</h4></li>
-                <li><div class="dot"></div></li>
-                <li><h4 class="feature_2">Back End Dev</h4></li>
-                <li><div class="dot"></div></li>
-                <li><h4 class="feature_2">2015</h4></li>
-            </ul>
             <div class="description item">
                 <p>
                     Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
                 </p>
             </div>
-            <h4 class="lets_connect item">
+            <h4 class="lets_connect_2 item">
                 <a>LET'S CONNECT</a>
             </h4>
             <section class="footer item">
                 <h2>.</h2>
-                <ul class="footer_ul">
+                <ul class="footer_ul_2">
```

Comment:
_**Figma Design**_
![original](https://user-images.githubusercontent.com/26856491/121662203-f3449880-ca9c-11eb-917e-5de520092db9.png)

_**Your portfolio**_
![clone](https://user-images.githubusercontent.com/26856491/121662211-f6d81f80-ca9c-11eb-9c96-b319000dd617.png)

Looking at the images above, I think you should:

- [ ] Let the first half (left side) of the section take 40% and the right one (right side) take 60% of the section's width, and the contents of each of them are aligned to the right.
- [ ] Adjust the width of the `Get My Resume` button so it does not occupy the entire width of the half.